cmd/dagger/client.go

```diff
@@ -14,3 +43,183 @@ var clientCmd = &cobra.Command{
 		"experimental": "true",
 	},
 }
+
+var clientInstallCmd = &cobra.Command{
+	Use:     "install [options] [path]",
```

Comment:
We might also want to alias to `use` as well, since that's aliased at the top-level as well.